owlplug-client/src/main/java/com/owlplug/core/controllers/OptionsController.java

```diff
@@ -200,12 +218,16 @@ public void refreshView() {
     storeSubDirectoryCheckBox.setSelected(this.getPreferences().getBoolean(ApplicationDefaults.STORE_SUBDIRECTORY_ENABLED, true));
     warningSubDirectory.setVisible(!this.getPreferences().getBoolean(ApplicationDefaults.STORE_SUBDIRECTORY_ENABLED, true));
     storeDirectoryCheckBox.setSelected(this.getPreferences().getBoolean(ApplicationDefaults.STORE_DIRECTORY_ENABLED_KEY, false));
+    storeByMakerCheckBox.setSelected(this.getPreferences().getBoolean(ApplicationDefaults.STORE_BY_MAKER_ENABLED_KEY, false));
```

Comment:
I've defaulted this to false, so the end-user must opt-in

Reply:
LGTM